app/models/card.rb

```diff
@@ -1,2 +1,15 @@
+class GoodnessValidator < ActiveModel::Validator
+  def validate(record)
+    if record.original_text? &&
+      record.translated_text? &&
+      record.original_text.downcase.strip.split(' ') == record.translated_text.downcase.strip.split(' ')
+      record.errors[:base] << "Слова не должны совпдать"
+    end
+  end
+end
+
 class Card < ActiveRecord::Base
+  validates :original_text, :translated_text, presence: {in: true,
+    message: "%{value} не может быть пустым"}
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Space inside } missing.